File: baekjoon/js/11399.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

const [, inputs] = buf.split("\n");
const times = inputs.split(" ").map(Number);

times.sort((a, b) => a - b);

let result = 0;
let acc = 0;

for (const time of times) {
  result += acc + time;
  acc += time;
}

console.log(result);

File: baekjoon/rust/11399.rs
use std::io::{read_to_string, stdin};

fn main() {
    let buf = read_to_string(stdin()).unwrap();
    let mut inputs: Vec<_> = buf
        .split_whitespace()
        .skip(1)
        .flat_map(str::parse::<u32>)
        .collect();

    inputs.sort();

    let mut result = 0;
    let mut acc = 0;

    for num in inputs {
        acc += num;
        result += acc;
    }

    print!("{}", result);
}
